interview-questions.md

```diff
@@ -8,50 +8,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: A conditional statement is a statement that gives step by step instructions to produce the desired outcome.
 
 Researched answer:
 
+Conditional statements can change depending on the conditions given. They control the output in javascript and determine whether the code will work or not.
+
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: A git is used to collaborate on projects with other coders. Github is there the main project is stored.
 
-Researched answer:
+Researched answer: A git is a system that allows collobarators to simultaneously work on projects at the same time. Each coder can add changes then push them up to Github. Github is a cloud based service that hosts the main projects in repositories, allowing the project to be better managed by all colaborators.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The loose equality operator and the strict equality operator
 
-Researched answer:
+Researched answer: Boolean values are returned when using the strict or loose equality operators. The loose equality operator converts the data type to make the equal first, then makes a comparison. The strict equality operator requires that the types of data be the same before returning a boolean.
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: An index is a placeholder for a character in a string
```

Comment:
Make sure to expand on your answers.  This is supposed to reflect how you would respond to an interviewer in an interview.  Short answers do not give you the chance to share your knowledge.  Try providing examples and digging deeper into the definitions.